### ISSUE TITLE
Don't use `-rc2`, never release with packages from an rc release

### ISSUE THREAD

**Issue by lewisdiamond** [tessel/openwrt-tessel#78]
https://github.com/tessel/openwrt-tessel/blob/d3052fbdee4cc8e99b0ae121f0b81867ceda8d30/config.mk#L78

**Comment by k45514n** [comment]
On it.

**Comment by k45514n** [comment]
This is going to take a bit more work-- the build system as a whole needs a few updates. I'll create an issue for that and update accordingly.

**Comment by bart** [comment]
I think this will also fix the `opkg update` issue, right? Because it wants to download files from `http://downloads.openwrt.org/chaos_calmer/15.05-rc2` which doesn't exist anymore.

**Comment by k45514n** [comment]
Yes, that is correct, that is part of what I am changing. I'll set up an in-progress PR and attach it here so y'all can see what I'm up to.
